src/azul/logging.py

```diff
@@ -163,20 +175,59 @@ def silenced_es_logger():
             assert es_log.level == original_log_level
 
 
-def http_body_log_message(body_type: str,
-                          body: bytes | bytearray | str | IO[bytes] | IO[str] | None,
+type BodyType = Literal['request', 'response']
+
+first_kibibit = 1024
```

Comment:
Brittle name. Generally, you don't want to have to change a variable name when you change its value.